plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -220,30 +225,23 @@ class ServiceActionDisabledModal extends React.Component {
 
   getServiceEditMessage() {
     const { intl } = this.props;
-    const command = this.getUpdateCommand();
 
     return (
-      <div>
-        <p>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1"
-          })}
+      <MountService.Mount
+        type={"ServiceEditMessage:Modal"}
+        getClipboardTrigger={this.getClipboardTrigger}
```

Comment:
Sorry, I should've been more clear with my last comment! I don't think the `getClipboardTrigger` and `getUpdateCommand` should live here. We shouldn't be passing commands into the MountService. Something is very wrong if we have to be doing that.